### PR TITLE
fix: prevent empty todos

### PR DESCRIPTION
**Pull request by corneliusroemer**
Empty todos are not meaningful, also they can't be deleted.

### CODE REVIEW

File: 01_todo_app/main.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from fasthtml.common import *
+from fasthtml.common import fast_app
```
Comment (jph00) [inline 01_todo_app/main.py:2]: Why did you add this?

Comment (corneliusroemer) [inline 01_todo_app/main.py:2]:
```suggestion
```

Because I'm stupid :D - probably some auto import from editor or something like that - sorry that I missed it.

**Comment by corneliusroemer** [comment]
I've removed the import

**Comment by jph00** [comment]
Thank you :)
